lib/ueberauth/strategy/slack.ex

```diff
@@ -159,7 +159,7 @@ defmodule Ueberauth.Strategy.Slack do
         auth: conn.private.slack_auth,
         token: conn.private.slack_token,
         user: conn.private.slack_user,
-        team: conn.private.slack_team
+        team: Dict.get(conn.private, :slack_team)
```

Comment:
These could actually all be turned into `conn.private[:slack_*]`.

The square bracket access will just return `nil` for missing keys.